locales/en-US/app.ftl

```diff
@@ -61,7 +61,7 @@ emailOptInInput =
 emailOptInButton = Sign me up
 emailOptInSkip = Skip
 emailOptInConfirmationTitle = Email Sent
-emailOptInSuccessMessage = When you get a chance, head over to your inbox to verify your email address. Thank you!
```

Comment:
@lmorchard @chuckharmston What do we want to change the ID to?
I'm guessing it should still have [at least] the `emailOptIn` prefix, but not sure what the best suffix is for this string.